general/auth/google.rst

```diff
@@ -1,49 +1,95 @@
+   .. image:: media/google-auth-create-project.png
+      :align: center
+      :alt: Create project button
 
-.. image:: media/google05.png
-    :align: center
+   .. image:: media/google-auth-new-project-details.png
+      :align: center
+      :alt: Filling in the details of a new project
 
-- Then click on **Add Credentials** and select the second option (OAuth 2.0 Client ID).
+   .. note::
+      - Type the name of your own company, NOT odoo.com as in the example.
 
-.. image:: media/google06.png
-    :align: center
+#. On the left side menu, click on :menuselection:`OAuth consent screen`.
 
-.. image:: media/google07.png
-    :align: center
+   .. image:: media/google-oauth-consent-selection.png
+      :align: center
+      :alt: Google oauth consent selection menu
 
-- Check that the application type is set on **Web Application**. Now configure the allowed pages on which you will be redirected.
+#. Choose one of the options **(Internal / External)** as instructed, and click on *Create*.
 
-- To achieve this, complete the field **Authorized redirect URIs**. Copy paste the following link in the box: http://mydomain.odoo.com/auth_oauth/signin. Then click on **Create**
+   .. image:: media/google-oauth-consent.png
+      :align: center
+      :alt: Choice of a user type in oauth consent
 
-.. image:: media/google08.png
-    :align: center
+#. Fill in your details, then click on *Save and Continue*.
 
-.. image:: media/google09.png
-    :align: center
+   .. image:: media/google-auth-edit-app-registration.png
+      :align: center
+      :alt: Edit app registration page
 
-- Once done, you receive two information (your Client ID and Client Secret). You have to insert your Client ID in the **General Settings**.
+   .. image:: media/google-auth-edit-app-registration-part2.png
+      :align: center
+      :alt: Edit app registration page part 2
+
+#. Click on *Save and Continue* once again, then click on *Back to Dashboard*.
+
+#. On the left side menu, click on :menuselection:`Credentials`.
+
+   .. image:: media/google-auth-credentials-button.png
+      :align: center
+      :alt: Credentials button menu
+
+#. Click on *Create Credentials* and select **OAuth client ID**.
+
+   .. image:: media/google-oauth-client-id.png
+      :align: center
+      :alt: Oauth client id selection
+
+#. Select **Web Application** as the Application type. Now configure the allowed pages on which you 
+   will be redirected. To achieve this, complete the field **Authorized redirect URIs**. Copy paste 
+   the following link into the box: http://mydomain.odoo.com/auth_oauth/signin (replace "mydomain"
+   with your own domain). Then click on *Create*.
+
+   .. image:: media/google-create-oauth-client-id.png
+      :align: center
+      :alt: Creating oauth client id
+
+#. Once done, you receive two infos (your Client ID and Client Secret). *Insert* your Client ID in 
+   the **General Settings** on Odoo.
```

Comment:
You don't describe at all how to do it in Odoo.
Also, the image below shows the "OAuth providers" menu and not the specific Google Authentication setting.

If divided in sections/paragraphs, you could have a part about how to retrieve it in Google and another on how to add it in Odoo.